packages/server/src/app/db/comments/comments.service.ts

```diff
@@ -56,10 +58,31 @@ export class CommentsService {
         });
     }
 
+    /**
+     * Creates a new comment that belongs to some content.
+     * 
+     * @param user A user's JWT payload
+     * @param contentId The content the comment belongs to
+     * @param commentInfo The comment's info
+     */
+    async createContentComment(user: any, contentId: string, commentInfo: models.CreateComment): Promise<documents.ContentCommentDocument> {
+        const newComment = new this.contentCommentModel({
+            user: user.sub,
+            contentId: contentId,
+            body: commentInfo.body
+        });
+
+        return await newComment.save().then(async doc => {
```

Comment:
don't use a `.then()` callback, they're old and gross. Just do 

```typescript
let doc = await newComment.save();
await this.contentService.addComment(contentId);
return doc;
```
It's semantically equivalent.